browser/widevine/widevine_utils.cc

```diff
@@ -119,26 +124,21 @@ void RequestWidevinePermission(content::WebContents* web_contents,
                    new WidevinePermissionRequest(web_contents, for_restart));
 }
 
-void DontAskWidevineInstall(content::WebContents* web_contents, bool dont_ask) {
-  Profile* profile = static_cast<Profile*>(web_contents->GetBrowserContext());
-  profile->GetPrefs()->SetBoolean(kAskWidevineInstall, !dont_ask);
-}
-
 void RegisterWidevineProfilePrefsForMigration(
     user_prefs::PrefRegistrySyncable* registry) {
-  registry->RegisterBooleanPref(kWidevineOptedIn, false);
+  registry->RegisterBooleanPref(kWidevineEnabled, false);
 }
 
 void RegisterWidevineLocalstatePrefs(PrefRegistrySimple* registry) {
-  registry->RegisterBooleanPref(kWidevineOptedIn, false);
+  registry->RegisterBooleanPref(kWidevineEnabled, false);
 }
 
 bool IsWidevineOptedIn() {
```

Comment:
this method name should change to match the pref name `IsWidevineEnabled`

Reply:
Fixed.